agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1853,6 +1853,8 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<sRadioInfo>
 
         soc->sta_iface.assign(request->sta_iface(message::IFACE_NAME_LENGTH));
         soc->hostap_iface.assign(request->hostap_iface(message::IFACE_NAME_LENGTH));
+        // Create a local copy on this process database instance. Will be removed on PPM-83 phase 5
```

Comment:
Maybe add a `TODO:` for these comments, to easily find them later on:
```suggestion
        // TODO: Create a local copy on this process database instance. Will be removed on PPM-83 phase 5
```

Reply:
These comments are for me. I'll remove them soon.